src/Microsoft.Data.Sqlite/SqliteDataReader.cs

```diff
@@ -9,6 +9,7 @@
 using System.Globalization;
 using Microsoft.Data.Sqlite.Interop;
 using Microsoft.Data.Sqlite.Utilities;
+using static Microsoft.Data.Sqlite.Interop.Constants;
```

Comment:
*I'm* fine with it. @Eilon do we (aspnet) have any guidance on this new language feature yet?